src/api/services/publishDocuments.spec.js

```diff
@@ -45,6 +45,9 @@ const getCtx = ({ subresources } = {}) => ({
     publishedDataset: {
         insertBatch: 'publishedDataset.insertBatch()',
     },
+    hiddenResource: {
+        findAll: jest.fn().mockImplementation(() => null),
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏You should probably return an empty array here instead of null, to make the mocked implementation more realistic